multiuser/machine-auth/che-multiuser-machine-authentication/src/main/java/org/eclipse/che/multiuser/machine/authentication/server/MachineLoginFilter.java

```diff
@@ -80,75 +74,46 @@ public void init(FilterConfig filterConfig) {}
   @Override
   public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
       throws IOException, ServletException {
-    final HttpServletRequest httpRequest = (HttpServletRequest) request;
-    final String token = tokenExtractor.getToken(httpRequest);
+
+    final String token = tokenExtractor.getToken((HttpServletRequest) request);
     if (isNullOrEmpty(token)) {
       chain.doFilter(request, response);
       return;
     }
-    // check token signature and verify is this token machine or not
     try {
-      HttpSession session = ((HttpServletRequest) request).getSession(true);
-      Subject sessionSubject = (Subject) session.getAttribute("che_subject");
-      if (sessionSubject == null || !sessionSubject.getToken().equals(token)) {
-        try {
-          sessionSubject = extractSubject(token);
-          session.setAttribute("che_subject", sessionSubject);
-        } catch (NotFoundException e) {
-          sendErr(
-              response,
-              SC_UNAUTHORIZED,
-              "Authentication with machine token failed because user for this token no longer exist.");
-          return;
-        }
-      }
-
-      try {
-        EnvironmentContext.getCurrent().setSubject(sessionSubject);
-        chain.doFilter(addUserInRequest(httpRequest, sessionSubject), response);
-      } finally {
-        EnvironmentContext.reset();
-      }
-    } catch (NotMachineTokenJwtException ex) {
-      // not a machine token, bypass
+      super.doFilter(request, response, chain);
+    } catch (NotMachineTokenJwtException e) {
+      // bypass - not our token
       chain.doFilter(request, response);
-    } catch (ServerException | JwtException e) {
-      sendErr(
+    } catch (JwtException e) {
+      sendError(
```

Comment:
log and error please

Reply:
same as previous - it's just an invalid user input, no big sense to log it.